src/controller/CHIPDeviceController.cpp

```diff
@@ -781,6 +781,19 @@ CHIP_ERROR DeviceCommissioner::GetAttestationChallenge(ByteSpan & attestationCha
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR DeviceCommissioner::GetAttestationChallenge(DeviceProxy * device, ByteSpan & attestationChallenge)
```

Comment:
Again, why is this an API on DeviceCommissioner?  It does not use the DeviceCommissioner in any way.